test/e2e/gardener/shoot/create_update_delete.go

```diff
@@ -40,15 +40,15 @@ var _ = Describe("Shoot Tests", Label("Shoot", "default"), func() {
 
 	// explicitly use one version below the latest supported minor version so that Kubernetes version update test can be
 	// performed
-	f.Shoot.Spec.Kubernetes.Version = "1.24.8"
+	f.Shoot.Spec.Kubernetes.Version = "1.25.4"
```

Comment:
Nit : `1.25.5`

Reply:
This comment is not resolved due to https://github.com/gardener/gardener/pull/7275#issuecomment-1381773647.